4.0.0/migration.md

```diff
@@ -19,6 +20,124 @@ The migration process is initiated by the Installer if the following requirement
 
 * MySQL stateful sets (HA) or deployments (no HA) are running
 
+As a preflight check, it's suggested to run a SQL procedure on a MySQL pod in order to check if there are constraint violations:
```

Comment:
Before you run the Installer, run a SQL procedure on a MySQL pod and check for constraint violations: